SampleBrowser/SampleBrowser/Extensions/WindowUUID+Extension.swift

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/// A unique identifier for a particular window in Firefox iOS.
+/// On iPad this UUID corresponds to an individual window which
+/// manages its own unique set of tabs. Multiple Firefox windows
+/// can be run side-by-side on iPad (once multi-window is enabled). [FXIOS-7349]
+public typealias WindowUUID = UUID
```

Comment:
I looked into that previously and ran into some problems, though I can't recall the specifics now (it was during the mega-refactor and follow-up PRs). I do think that -- most likely -- we should not duplicate this extension/typealias over into the SampleBrowser for now. It shouldn't really a problem for us to just use `UUID` in SampleBrowser for the time being, especially since the notion of multiple windows doesn't really apply there.